components/announcement_bar/renewal_link/renewal_link.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+
+import {Client4} from 'mattermost-redux/client';
+import {FormattedMessage} from 'react-intl';
+
+import {trackEvent} from 'actions/telemetry_actions';
+import {
+    ModalIdentifiers,
+} from 'utils/constants';
+
+import NoInternetConnection from '../no_internet_connection/no_internet_connection';
+
+export interface RenewalLinkProps {
+    actions: {
+        openModal: (modalData: { modalId: string; dialogType: any; dialogProps?: any }) => void;
+    };
+}
+
+const RenewalLink: React.FC<RenewalLinkProps> = (props: RenewalLinkProps) => {
+    const [renewalLink, setRenewalLink] = useState('https://mattermost.com/renew/');
```

Comment:
Not sure if we still need to default to this url, this is now being replaced with the new renewal link returned by the server.